src/me/mrCookieSlime/Slimefun/api/TickerTask.java

```diff
@@ -202,20 +205,27 @@ public void info(CommandSender sender) {
 			tellraw.addText("   &7&oHover for more Info");
 			StringBuilder hover = new StringBuilder();
 			int hidden = 0;
-			
-			for (Map.Entry<String, Integer> entry: machineCount.entrySet()) {
-				long timings = machineTimings.get(entry.getKey());
-				if (timings > 0) hover.append("\n&c").append(entry.getKey()).append(" - ").append(entry.getValue()).append("x &7(").append(timings).append("ms)");
-				else hidden++;
+
+			Map<String, Long> timings = machineCount.entrySet().stream()
+					.sorted(Comparator.comparingLong(machineTimings::get).reversed())
+					.collect(Collectors.toMap(Map.Entry::getKey, machineTimings::get, (e1, e2) -> e1, LinkedHashMap::new));
+
+			for (Map.Entry<String, Long> entry : timings.entrySet()) {
+				if (entry.getValue() > 0)
+				    hover.append("\n&c").append(entry.getKey()).append(" - ")
+						.append(entry.getValue()).append("x &7(").append(entry.getValue()).append("ms, ")
```

Comment:
entry.getValue() no longer refers to the value it is supposed to refer to.

Reply:
Yep noticed and fixed that already